app/helpers/case_contacts_helper.rb

```diff
@@ -8,6 +8,10 @@ def duration_minutes(case_contact)
     case_contact.duration_minutes.to_i.remainder(60)
   end
 
+  def set_contact_made_false(case_contact)
+    case_contact.persisted? && case_contact.contact_made == false
```

Comment:
Thanks for catching this. The `persisted?` behavior was subtle, so I thought it was unnecessary when I removed it. I was obviously wrong :)